test/Engine/ProtoTest/MultiLangTests/NullInputTest.cs

```diff
@@ -0,0 +1,26 @@
+using NUnit.Framework;
+using ProtoCore.DSASM.Mirror;
+namespace ProtoTest.MultiLangTests
+{
+    class NullInputTest : ProtoTestBase
+    {
+        public override void Setup()
+        {
+            base.Setup();
+
+            // TODO: Determine if this is necessary for this test
+            FFITarget.DisposeCounter.Reset(0);
+        }
+
+        [Test]
+        public void DYN_2093_NullInput_Flatten()
+        {
+            var code =
+@"
+import(""DSCoreNodes.dll"");
```

Comment:
ah, I see what you mean now about the import - this is a guess - but perhaps if you don't do the import, the test does not fail because the function is simply not found and that results in an error and not a crash.

Reply:
Thanks for the reply on the question. That is pretty much what I was thinking was going on, but just wanted a second opinion. The thing about it was if this is what is happening it means that not having the import caused a test to succeed where it should have failed and there was no immediate indication that that was what was going on. It makes me a little worried if there could be other cases like this now or in the future.

Reply:
Possibly we need to have some method to surface the failing import issue in tests more visibly.

Reply:
well, to write a more robust test, maybe we can assert a value for a function which runs after the list.flatten call - 

somethingLike

```
import(""DSCoreNodes.dll"");
testData = [[[1]]];
DSCore.List.flatten(null);
result = DSCore.List.flatten(testData); // assert that result = [1] 
```

it's kind of unclear I guess to someone reading it later, but it should check that the execution did not crash?

Reply:
@ColinDayOrg - hmm! - I'm not sure if the missing method error will throw an exception - but you could try wrapping the 
`thisTest.RunScriptSource(code);`

call in
```
 Assert.DoesNotThrow( ()=>{ thisTest.RunScriptSource(code);
}
```

Reply:
As it is now the test fails without the code changes and passes with the changes, so I don't think that there necessarily needs to be additional code for this specific test. I am trying to find other cases that can be tested to be more robust. I just wanted to point out that it appears that there can be test cases like this that could be created that would pass even though they should be failing and there was no indication about what was going on. I can add these changes you were discussing above if you think it would be better.

Reply:
I will though give your suggestions a try and see if it clarifies the situation.